3rd/src/031Exception.py

```diff
@@ -0,0 +1,18 @@
+#! python3
+
+#エラーが発生した際に別処理へ遷移させたい箇所をtry～exceptで囲む。
+#基本的に全部でいい。
+try:
+
+#例文として存在しないファイルを指定
+#ファイル名を間違ったりして存在しないファイルを指定するとエラーになります。
+    output = open("exception.txt", "r",encoding="utf-8")
+
+    for o in output:
+        print (o);
+        output.close()

```

Comment:
```suggestion
    output.close()
```
for節の中にclose()があるので外に出した方がいいと思います。
その前に例外が補足されるので処理が行われませんが。。。

Reply:
対応します。